components/butterfly/src/protocol/swim.rs

```diff
@@ -15,3 +15,48 @@
 include!("../generated/butterfly.swim.rs");
 
 pub use self::{membership::Health, swim::Payload as SwimPayload, swim::Type as SwimType};
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    // These tests assure that we can properly compare Health values
+    // along the spectrum of
+    //
+    //   Alive -> Suspect -> Confirmed -> Departed
```

Comment:
Since `Health` implements the `PartialOrd` trait, isn't is enough to assert

1. Alive < Suspect
2. Suspect < Confirmed
3. Confirmed < Departed

?

These tests are certainly thorough, but if we wanted to add another state or change something, it seems to require more effort than it needs to.

Reply:
Derp... yes.